PluralKit.Bot/Commands/Message.cs

```diff
@@ -13,6 +13,8 @@
 
 using NodaTime;
 
+using IMetrics = App.Metrics.IMetrics;
```

Comment:
I think you can just `using App.Metrics;`?

Reply:
Doing that makes the compiler yell because `App.Metrics` has an incompatible `IClock` definition (which is pulled in by `using NodaTime;`)

Reply:
in that case I'd change to use `SystemClock.Instance` instead of the DI IClock, but it's probably irrelevant now since the code's getting moved to ProxyService anyway

Reply:
taking a quick look at this, removing the DI IClock (and the `NodaTime` import) would require rewriting unrelated stuff around the `pk;edit` timeout, and we don't really wanna mess with that in this PR tbh!

Reply:
oh, that's fine. I'll fix it myself when I merge the PR then